README.md

```diff
@@ -16,13 +16,24 @@ Seamless integration with any Nano compatible RPC backend/websocket and the aim
 ___
 
 ## How To Use
+Nault comes in different flavors to suit your need.
+#### Desktop App
+Available for Windows/Mac/Linux - just head over to the [latest release](https://github.com/Nault/Nault/releases/latest) and download the version for your OS.
 
-Nault is available on your desktop (Windows/Mac/Linux) - just head over to the [latest release](https://github.com/Nault/Nault/releases/latest) and download the version for your OS.
+If you want to verify the binary checksum there are plenty of apps to do this and one way is using a powershell or bash terminal using sha-512.
 
+* **Powershell:** Get-FileHash -Path '.\Nault Setup x.x.x.exe' -Algorithm SHA512
```

Comment:
The name of the exe has dashes not spaces e.g.: https://github.com/Nault/Nault/releases/download/v1.8.6/Nault-Setup-1.8.6.exe